<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Oriented</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TopologicalOrder">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>TopologicalOrder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findRootButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>520</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>FindRoot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAll">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearAll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearEdges">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearEdges</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectedComponents">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ConnectedComponents</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StrongConnectedComponents">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>StrongConnectedComponents</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Undo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
